tests/page_objects/providers_care_platform_page.yaml

```diff
@@ -61,8 +61,8 @@ providers_facilities_page:
   devices_button: //button[text()='Devices']
   tabs_content: //div[@class='tabs-content']
   title_subtab_devices: //h4[text()='Facility Devices']
-  call_based_on_device_button: //strong[text()='Device $AND_CLI_facility_devices_user1_number$:']/ancestor::tr//a[contains(.,'Call')]
-  message_based_on_device_button: //strong[text()='Device $AND_CLI_facility_devices_user1_number$:']/ancestor::tr//a[contains(.,'Message')]/button
+  call_based_on_device_button: //strong[text()='Device $AND_CLI_facility_devices_user1_number$:']/ancestor::tr//a[contains(@class,'button button-basic text-xs')]
```

Comment:
Could you make the selector more relative to the object function or text, instead of a class based on style?

Reply:
thank you, fixed it on the latest commit